src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -177,61 +194,89 @@ doclet.help.class_interface.declaration=\
+doclet.help.class_interface.enum=\
+    Only enum classes have enum constants.
+doclet.help.class_interface.record=\
+    The components of a record class are displayed as part of the declaration of the record class.
+doclet.help.class_interface.property=\
+    Properties are a feature of JavaFX.
+doclet.help.other_files.head=\
+    Other Files
+doclet.help.other_files.body=\
+    Packages and modules may contain pages with additional information related to the \
+    declarations nearby.
 doclet.help.use.head=\
     Use
 doclet.help.use.body=\
     Each documented package, class and interface has its own Use page.  This page describes what \
     packages, classes, methods, constructors and fields use any part of the given class or \
-    package. Given a class or interface A, its "Use" page includes subclasses of A, fields declared \
+    package. Given a class or interface A, its Use page includes subclasses of A, fields declared \
     as A, methods that return A, and methods and constructors with parameters of type A.  \
     You can access this page by first going to the package, class or interface, then clicking on \
-    the "Use" link in the navigation bar.
+    the USE link in the navigation bar.
 doclet.help.tree.head=\
     Tree (Class Hierarchy)
+# 0: link to main Class Hierarchy page;  1: java.lang.Object
 doclet.help.tree.intro=\
     There is a {0} page for all packages, plus a hierarchy for each package. Each hierarchy page \
     contains a list of classes and a list of interfaces. Classes are organized by inheritance \
     structure starting with {1}. Interfaces do not inherit from {1}.
 doclet.help.tree.overview=\
-    When viewing the Overview page, clicking on "Tree" displays the hierarchy for all packages.
+    When viewing the Overview page, clicking on TREE displays the hierarchy for all packages.
 doclet.help.tree.package=\
-    When viewing a particular package, class or interface page, clicking on "Tree" displays the \
+    When viewing a particular package, class or interface page, clicking on TREE displays the \
     hierarchy for only that package.
+# 0: link to the Deprecated summary page
 doclet.help.deprecated.body=\
     The {0} page lists all of the API that have been deprecated. A deprecated API is not \
     recommended for use, generally due to shortcomings, and a replacement API is usually given. \
     Deprecated APIs may be removed in future implementations.
+# 0: link to the Preview summary page
 doclet.help.preview.body=\
     The {0} page lists all of the Preview APIs. \
     Preview APIs may be removed in future implementations.
 doclet.help.index.head=\
     Index
+# 0: link to Index page;  1: list of links to pages
 doclet.help.index.body=\
     The {0} contains an alphabetic index of all classes, interfaces, constructors, methods, \
-    and fields, as well as lists of all packages and all classes.
+    and fields in the documentation, as well as summary pages such as {0}.
```

Comment:
The link at the end of the sentence is the same as the one at the beginning of the sentence, which is the main index page. I think you probably meant to link to one of the summary pages.

Reply:
Thanks: this is a good illustration of why the preceding comment is useful. :-)